<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanner</class>
 <widget class="QMainWindow" name="scanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="hostnameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Scan</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resultBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portsEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>115</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>115</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Ports</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Hostname</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
